r/R/arrow-package.R

```diff
@@ -38,27 +38,33 @@ supported_dplyr_methods <- list(
   select = NULL,
   filter = NULL,
   collect = NULL,
-  summarise = NULL,
+  summarise = c(
+    "window functions not currently supported;",
+    'arguments `.drop = FALSE` and `.groups = "rowwise" not supported'
+  ),
   group_by = NULL,
   groups = NULL,
   group_vars = NULL,
   group_by_drop_default = NULL,
   ungroup = NULL,
-  mutate = NULL,
+  mutate = c(
+    "window functions (e.g. things that require aggregation within groups)",
```

Comment:
```suggestion
    "window functions (i.e. things that require aggregation within groups)",
```
nit, but only if window functions are literally defined by this, which I wasn't sure about